src/Lykke.AlgoStore.Api/Infrastructure/AutoMapperProfile.cs

```diff
@@ -41,6 +44,15 @@ public AutoMapperProfile()
                 .ForMember(dest => dest.BackTestTradingAssetBalance, opt => opt.Ignore())
```

Comment:
We don't need these properties in AlgoInstanceModel, because we don't store them in Algo Instance database, we need them only in Statistics
We may need to refactor the service logic.